tests/phpunit/tests/theme/wpThemeJson.php

```diff
@@ -3210,7 +3217,6 @@ public function test_get_property_value_valid() {
 	 *
 	 * @param array $styles An array with style definitions.
 	 * @param array $path   Path to the desired properties.
-	 *
```

Comment:
Note: These changes are already included in another PR, #5944. If that PR is merged before this one, we need to update this PR accordingly and remove these changes.

Reply:
👍 Thanks for spotting that. I missed phpcbf had removed it in both PRs. It's been removed here only (https://github.com/WordPress/wordpress-develop/pull/5942/commits/8a647662eda9ff3b8f895089138db9f970da158a)